akka-docs/src/main/paradox/scala/stream/stream-io.md

```diff
@@ -110,6 +110,19 @@ of the server, and `q` in the case of the client. This is implemented by @scala[
 and concatenating a `Source` with a single `BYE` element which will then be sent after the original source completed]@java[using a custom `GraphStage`
 which completes the stream once it encounters such command].
 
+### Using framing in your protocol
+
+Network connections just pass streams of bytes, and does not know what is a logical chunk of bytes from the
```

Comment:
Maybe better: "Streaming transport protocols like TCP just pass ...".

"Network connections" seem to be much more general than that.